clang/include/clang/Driver/Options.td

```diff
@@ -6488,6 +6488,9 @@ defm stack_arrays : BoolOptionWithoutMarshalling<"f", "stack-arrays",
 defm loop_versioning : BoolOptionWithoutMarshalling<"f", "version-loops-for-stride",
   PosFlag<SetTrue, [], [ClangOption], "Create unit-strided versions of loops">,
    NegFlag<SetFalse, [], [ClangOption], "Do not create unit-strided loops (default)">>;
+
+def fcuda : Flag<["-"], "fcuda">, Group<f_Group>,
```

Comment:
I wonder if we also want to wire it to clang's `-x cuda` option.  It might be good to use the same flags for clang/flang in makefiles/cmake.  Though, I am not sure if the meaning of the two options is exactly the same.  Maybe someone else knows.

Reply:
I don't quite understand what actually `-fcuda` enables. Could the flag be more descriptive and the help text expanded? Also, what's the equivalent in Clang? We ought to keep both drivers in sync.

In particular, if this is something specific to Flang then I would avoid generic names like `-fcuda`.

Reply:
The alternative is to use the same as `clang` with `-x <language>`. I will need to check if we can filter the language so only the ones meaningful to flang can be passed.

Reply:
I switch to use the `-x cuda` option which has a similar meaning.